web-common-marlowe/src/Marlowe/Extended.purs

```diff
@@ -309,6 +321,16 @@ instance fillableValue :: Fillable Value TemplateContent where
     go :: forall a. (Fillable a TemplateContent) => a -> a
     go = fillTemplate placeholders
 
+instance valueHasParties :: HasParties Value where
+  getParties (AvailableMoney accId _) = getParties accId
+  getParties (NegValue val) = getParties val
+  getParties (AddValue lhs rhs) = getParties lhs <> getParties rhs
+  getParties (SubValue lhs rhs) = getParties lhs <> getParties rhs
+  getParties (Scale _ val) = getParties val
+  getParties (ChoiceValue choId) = getParties choId
+  getParties (Cond obs lhs rhs) = getParties obs <> getParties lhs <> getParties rhs
+  getParties _ = Set.empty
```

Comment:
I prefer to specify all the clauses (not using catch-all underscores) because that way if we modify Marlowe types we will get an error here. As it is now, we will get a default behaviour of no parties which is harder to notice. But this is just my opinion, we can leave it as is, no big deal

Reply:
Okay, I don't have strong feelings on this so I'll go with your preference.